Week1/LongestSubString.java

```diff
@@ -0,0 +1,40 @@
+import java.util.HashSet;
+
+public class LongestSubString {
+    public static int subString(String input){
+
+    	/* Invalid Input testing */
+    	if (input.contains(" "))
+    	{
+    		return 0; //Zero Length if it is empty string or contains space.
+    	}
+    		
+    	String LongestStr = "";

```

Comment:
Java coding conventions are for local variables to be in camelCasing, rather than PascalCasing that you have here